### DISCUSSION TITLE
Unable to establish SSL connection.

### DISCUSSION THREAD

**Discussion by DrGratz** [Security-Onion-Solutions/securityonion#9130] [Unsupported Versions]
Hey! so I don't know much about this kind of stuff I'm trying to help a friend because it all somewhat seemed self explanatory but I ran into a problem after I got security onion all set up on my VM and after I looked into it more I realized that I messed up at the networking part at some point so now I'm trying to follow the videos exactly the same. They were easy to follow except I couldnt get wget to work, so I found out I had to download the wget.exe file and gnupg. however now I have a new problem I cant find anywhere for windows 10. I have an error saying 'gpg: no valid OpenPGP data found.'. And I have no idea what to do I was seeing something about curl but idk what that is either I just really need some help with this.

Edit1: Just a heads up any commands using apt-get or apt-key, I can't get to work and as far as I'm reading you can't use those on windows.

**Comment by dougburks** [comment]
I assume you're looking at the steps listed at https://securityonion.net/download. If that's the case, please notice this sentence:
```
For example, here are the steps you can use on most Linux distributions to download and verify our Security Onion ISO image.
```
So those steps are designed to work on Linux not Windows. 

The main point of https://securityonion.net/download is that you need to download the ISO image and verify it:
1. You can download the ISO image using wget, curl, or any standard web browser. 
2. To verify the ISO image, it's best to verify the GPG signature and you can download tools for Windows to help you with that. If you can't get GPG verification to work, then you can just verify the MD5/SHA1/SHA256 checksum listed at the top of the page using Powershell or other tools.
